classes/shortcodes.php

```diff
@@ -216,20 +252,28 @@ function do_shortcode( $args ) {
 					switch ( $tax_format ) {
 						case 'csv':
 						default:
-							return ! empty( $term_names ) ? $args['before'] . implode( ', ', $term_names ) . $args['after'] : '';
+							return ! empty( $term_names ) ? $args['before']
+							                                . implode( ', ',
+									$term_names ) . $args['after'] : '';
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Multi-line function call not indented correctly; expected 60 spaces but found 36
* Closing parenthesis of a multi-line function call must be on a line by itself

<!-- Comment by [SideCI](https://sideci.com) -->